### PR TITLE
Remove deprecated service list_events in favor of get_events

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Deprecate service list_events in favor of get_events

- The service `calendar.list_events` is deprecated in favor or `calendar.get_events`
- in 2023.12. Scheduled to be removed in 2024.6
- This PR adds docs for `get_events`
- This PR removes the docs of the deprecated `calendar.list_events` 
- Unlike list_events, get_events supports multiple responses
- This PR belongs into current because the parent has already been deployed.
- related parent PR in core: https://github.com/home-assistant/core/pull/102481

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/calendar.markdown [hunk 1]
```diff
@@ -194,6 +194,8 @@ data:
 
 ### Service `calendar.list_events`
 
+This service has been **deprecated**. It will be removed in a future release of Home Assistant. Please use `calendar.get_events` instead.
```
Comment (frenck) [inline source/_integrations/calendar.markdown]: We should just remove the deprecated service from the documentation.

**Review by mikeodr — APPROVED** [review]

**Comment by c0ffeeca7** [comment]
Thank you, @mikeodr

File: source/_docs/scripts/service-calls.markdown [hunk 2]
```diff
@@ -133,13 +133,12 @@ Examples of service response data are upcoming calendar events for the next week
...
 a `response_variable`. This is the [variable](/docs/scripts/#variables)
-that contains the response data. You can define any name for your `response_variable`. This example calls a service and stores the response in 
-the variable called `agenda`.
+that contains the response data. You can define any name for your `response_variable`. This example calls a service and stores the response in the variable called `agenda`.
 
 {% raw %}
 
 ```yaml
-service: calendar.list_events
+service: calendar.get_events
```
Comment (MartinHjelmare) [inline source/_docs/scripts/service-calls.markdown:141]: We should update the notify.gmail example below too as it consumes the response variable set in this example.

```yaml
service: notify.gmail_com
data:
  target: "gduser1@workspacesamples.dev"
  title: "Daily agenda for {{ now().date() }}"
  message: >-
    Your agenda for today:
    <p>
    {% for event in agenda['calendar.school'].events %}
    {{ event.start}}: {{ event.summary }}<br>
    {% endfor %}
    </p>
```

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**